### PR TITLE
fix: update start.go by adding descriptions and examples

### PR DESCRIPTION
**Pull request by ZouRui89**
**1.Describe what this PR did**
update start.go by add descriptions and examples
**2.Does this pull request fix one issue?** 

None
**3.Describe how you did it**
..
**4.Describe how to verify it**
try the process on my centos7
**5.Special notes for reviews**

### CODE REVIEW

File: cli/start.go [hunk 1]
```diff
@@ -10,8 +10,10 @@ import (
 )
 
 // startDescription is used to describe start command in detail and auto generate command doc.
-// TODO: add description
-var startDescription = ""
+var startDescription = "Start a created container object in Pouchd. " +
+	"When starting, the relevant resource preserved during creating period comes into use." +
+	"This is useful when you wish to start a container which has been created in advance" +
```
Comment (allencloud) [inline cli/start.go]: Please add . after `advance`.

**Comment by ZouRui89** [comment]
oops...my bad

**Comment by allencloud** [comment]
Thanks for your contribution. @ZouRui89 
Actually we wish that you could sign off your commits. First make sure you have set `git config user.name "Zou Rui"` and `git config user.email "zourui@xxx.com"`
Then use  `git commit -s -m "xxxxxx"` to commit and  push back to github.

In addition, we suggest you could use `git commit -s --amend` to amend commits, then in a PR there will be only one commit. 

Feel free to tell us if you need any help.

**Comment by ZouRui89** [comment]
Thx. My prs will be more standardized in the future. Grateful for your sharing.

**Comment by ZouRui89** [comment]
Do I have to revise this pr before it can be merged?

**Comment by allencloud** [comment]
LGTM. Maybe next time when you submit a new PR. 😄

**Pull request merged**